sympy/sets/tests/test_sets.py

```diff
@@ -249,9 +249,11 @@ def test_intersect1():
         (S.Naturals, S.Naturals0))
     assert all(i.intersection(S.Integers) is i for i in
         (S.Naturals, S.Naturals0))
-    s =  S.Naturals0
-    assert S.Naturals.intersection(s) is s
-    assert s.intersection(S.Naturals) is s
+    s = S.Naturals
+    assert S.Naturals0.intersection(s) is s
+    assert s.intersection(S.Naturals0) is s
+    assert S.Naturals.intersect(s) is s
+    assert S.Naturals0.intersect(S.Naturals0) is s
```

Comment:
Should this not be S.Naturals0?

I think these tests would be clearer without the local s e.g.:
```
assert S.Naturals0.intersect(S.Naturals0) is S.Naturals
```